src/Nethermind/Nethermind.Blockchain.Test/TransactionSelectorTests.cs

```diff
@@ -160,7 +161,7 @@ public static IEnumerable EnoughShardBlobTransactionsSelectedTestCases
                     tx.MaxFeePerBlobGas = 1;
                 });
                 maxTransactionsSelected.Transactions[1].BlobVersionedHashes =
-                    new byte[Eip4844Constants.MaxBlobGasPerTransaction / Eip4844Constants.GasPerBlob - 1][];
+                    new byte[5][];
```

Comment:
can you use values from spec instead of hardcoding?
also add these config value to release spec - Cancun.Instance